README.md

```diff
@@ -21,6 +21,25 @@ Commit messages start with the following identifiers enclosed in brackets:
 * feature/liquibase - configuration of the Liquibase structures
 * feature/intest - configuration of the separate module for Integration Tests
 
+
+### 2. Database initialization
+When running the app for the first time, prepare the database to work with.
+Run the SQL statements included in 'database/src/main/resources/init/users.sql', use a user with the DBA role.
+
+### 3. Liquibase changes
+Run the following command to update changes in your database structures:
+```shell
+./gradlew update
+```
+SQL files for step '0' in 'database/src/main/resources/changes' intentionally left blank.
+On this basis add next revisions of the database with db operations defined in the sql files in this directory.
+Each revision should be noted in the 'database/src/main/resources/changelog.yml' file.
+
+💡
+<small>Running the ```./gradlew updateSQL``` command before the above you can generate SQL scripts reflecting 
```

Comment:
How about:

The command ./gradlew updateSQL serves as a dry run variant of the update command. Instead of directly implementing the SQL instructions on the database, it outputs these instructions to the console. This feature proves useful when there's a desire to manually review or execute the scripts, such as through an SQL console.

Reply:
Changed